doc/nrf/dev_model_and_contributions/code_base.rst

```diff
@@ -151,28 +151,28 @@ The |NCS| version numbers are assigned based on very specific criteria.
 They follow the ``MAJOR.MINOR.PATCH`` pattern (also further called as ``X.Y.Z``), with possible additional postfixes depending on the release.
 All releases are accompanied by their respective :ref:`release_notes`.
 
-+----------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
-|        Release type        |                              Release criteria                              |                                                 Comments                                                                             |
```

Comment:
The table header is misleading as what is in the second column are not release criteria. Please merge the "Release and comments" into a single column named "Description".

Reply:
Rename the first column into "Version"

Reply:
Added.